src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c

```diff
@@ -1,10 +1,11 @@
 /*
  * File: z_shot_sun.c
  * Overlay: ovl_Shot_Sun
- * Description:
+ * Description: Leftover sun hitbox from OOT
```

Comment:
Other types can also spawn fairies, which it looks like would still function, so worth mentioning that too.